File: backend/src/main/java/com/school/exhibition/modules/profile/dto/ProfileDTO.java
package com.school.exhibition.modules.profile.dto;

import lombok.Data;

import java.time.LocalDate;
import java.util.List;

@Data
public class ProfileDTO {
    private Long id;
    private Long userId;
    private String userName;
    private String avatarUrl;
    private Long collegeId;
    private String collegeName;
    private String major;
    private Integer graduationYear;

    private String title;
    private Integer category;
    private String description;
    private String coverUrl;
    private LocalDate achieveDate;
    private String achieveLevel;
    private String issuingOrg;

    private Integer status;
    private String rejectReason;
    private Integer isOnShelf;
    private Integer displayWeight;
    private Integer viewCount;

    private List<MediaDTO> mediaList;
    private List<String> tags;
    private java.time.LocalDateTime createdAt;
    private java.time.LocalDateTime updatedAt;
}

File: unity-display/Assets/Scripts/DTOs/ProfileDTO.cs
using System;
using System.Collections.Generic;

namespace Exhibition.Network
{
    [Serializable]
    public class ApiResult<T>
    {
        public int code;
        public string msg;
        public T data;
        public bool IsSuccess => code == 200;
    }

    [Serializable]
    public class PageResult<T>
    {
        public long total;
        public List<T> records;
    }

    [Serializable]
    public class ProfileDTO
    {
        public long id;
        public long userId;
        public string userName;
        public string avatarUrl;
        public string collegeName;
        public string major;
        public int graduationYear;
        public string title;
        public int category;          // 1荣誉 2作品 3成绩
        public string description;
        public string coverUrl;
        public string achieveDate;
        public string achieveLevel;
        public string issuingOrg;
        public List<MediaDTO> mediaList;
        public List<string> tags;
    }

    [Serializable]
    public class MediaDTO
    {
        public long id;
        public int mediaType;          // 1图片 2视频
        public string fileUrl;
        public string thumbnailUrl;
        public int duration;
    }

    [Serializable]
    public class FaceRecognizeResult
    {
        public bool matched;
        public long userId;
        public string userName;
        public float score;
        public List<ProfileDTO> profiles;
    }

    [Serializable]
    public class FaceRecognizeRequest
    {
        public string deviceId;
        public string featureBase64;
    }
}